src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -903,6 +903,35 @@ function an_element(P::AbstractPolyhedron{N}) where {N<:Real}
     return σ(e₁, P)
 end
 
+"""
+    isbounded(P::AbstractPolyhedron{N}; solver=LazySets.default_lp_solver(N)) where {N<:Real}
+
+Determine whether a polyhedron is bounded.
+
+### Input
+
+- `P`       -- polyhedron
+- `backend` -- (optional, default: `default_lp_solver(N)`) the backend used
+               to solve the linear program
+
+### Output
+
+`true` iff the polyhedron is bounded
+
+### Algorithm
+
+We first check if the polyhedron has more than `max(dim(P), 1)` constraints,
+which is a necessary condition for boundedness.
+
+If so, we check boundedness via [`_isbounded_stiemke`](@ref).
+"""
+function isbounded(P::AbstractPolyhedron{N}; solver=LazySets.default_lp_solver(N)) where {N<:Real}
+    if length(P.constraints) <= max(dim(P), 1)
```

Comment:
Not every `AbstractPolyhedron` has a field `constraints`.
But computing the constraints twice sounds expensive in general. I suggest to only apply this heuristics to `HPoly`s. Action points:
* Add another dispatch method `isbounded(P::HPoly, ...)`.
* Update the docstring.
* Add a test for non-`HPoly` polyhedron.

Reply:
ok. i'll change it to use to use `constraints_list(P)` here and pass an HPolyhedron to the new algorithm Stiemke.

Reply:
> Add a test for non-`HPoly` polyhedron.

This might actually be impossible because all other types probably already define `isbounded`.